dojo/models.py

```diff
@@ -3838,6 +3838,18 @@ def merge_notifications_list(cls, notifications_list):
 
         return result
 
+    def __str__(self):
+        return f"Notifications about {self.product or 'all projects'} for {self.user}"
```

Comment:
`self.user` might be `None` (for system [non-user] notifications)

Reply:
Fixed by using "system notifications" phrase instead of the user string representation:

![image](https://github.com/DefectDojo/django-DefectDojo/assets/981572/2bd5b53d-4069-4b75-9395-0bfb760814c4)